po/zh-CN.po

```diff
@@ -8409,17 +8449,29 @@ msgid ""
 "reference of some kind, instead of storing a value directly.   \n"
 "    "
 msgstr ""
+"* 如果这里未使用 `Box`，且我们曾尝试将一个 `List` 直接嵌入 `List`，\n"
+"编译器就不会计算内存中结构体的固定大小，结构体看起来会像是无限大。\n"
+"\n"
+"* `Box` 大小与一般指针相同，并且只会指向堆中的下一个 `List` 元素，\n"
+"因此可以解决这个问题。\n"
+"\n"
+"* 将 `Box` 从 List 定义中移除后，画面上会显示编译器错误。如果您看"
+"到“Recursive with indirection”错误消息，这是在提示您使用 Box 或其他类型的引"
+"用，而不是直接储存值。\n"
+"    "
 
 #: src/std/box-niche.md:1
 msgid "# Niche Optimization"
-msgstr ""
+msgstr "# 小众优化"
```

Comment:
```suggestion
msgstr "# 利基优化"
```

Reply:
I didn't find the translation in question at https://rustwiki.org/zh-CN waiting for more people to discuss it